Suppress warning "Use FragmentContainerView instead of fragment tag"

## Kind (Required)

- Improvement

 ## Overview (Required)

- Use FragmentContainerView instead of fragment tag in activity_main.xml

<img width="622" alt="Screen Shot 2020-01-18 at 14 37 37" src="https://user-images.githubusercontent.com/376376/72659271-40375300-3a00-11ea-92b2-8a57282ac3c9.png">

 ## Links	

- https://developer.android.com/reference/androidx/fragment/app/FragmentContainerView

replaced and then getting error.

```
2020-01-18 14:47:47.682 8484-8484/io.github.droidkaigi.confsched2020.debug E/AndroidRuntime: FATAL EXCEPTION: main
    Process: io.github.droidkaigi.confsched2020.debug, PID: 8484
    java.lang.RuntimeException: Unable to start activity ComponentInfo{io.github.droidkaigi.confsched2020.debug/io.github.droidkaigi.confsched2020.MainActivity}: java.lang.IllegalStateException: Activity io.github.droidkaigi.confsched2020.MainActivity@1a5433f does not have a NavController set on 2131362199
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2913)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3048)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:78)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1808)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6669)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
     Caused by: java.lang.IllegalStateException: Activity io.github.droidkaigi.confsched2020.MainActivity@1a5433f does not have a NavController set on 2131362199
        at androidx.navigation.Navigation.findNavController(Navigation.java:61)
        at io.github.droidkaigi.confsched2020.MainActivity$navController$2.invoke(MainActivity.kt:83)
        at io.github.droidkaigi.confsched2020.MainActivity$navController$2.invoke(MainActivity.kt:68)
        at kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:74)
        at io.github.droidkaigi.confsched2020.MainActivity.getNavController(Unknown Source:7)
        at io.github.droidkaigi.confsched2020.MainActivity.setupNavigation(MainActivity.kt:153)
        at io.github.droidkaigi.confsched2020.MainActivity.onCreate(MainActivity.kt:93)
        at android.app.Activity.performCreate(Activity.java:7136)
        at android.app.Activity.performCreate(Activity.java:7127)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1271)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2893)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3048) 
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:78) 
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108) 
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1808) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loop(Looper.java:193) 
        at android.app.ActivityThread.main(ActivityThread.java:6669) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858) 
```

--- a/android-base/src/main/res/layout/activity_main.xml
+++ b/android-base/src/main/res/layout/activity_main.xml
@@ -48,6 +48,7 @@
                 app:titleTextColor="@{ContextsKt.getThemeColor(context, isIndigoBackground ? R.attr.colorOnPrimarySurface : R.attr.colorOnSurface)}"
                 />
 
+            <!-- Ignore because FragmentContainerView bug. See https://github.com/DroidKaigi/conference-app-2020/issues/378 -->
             <fragment
                 android:id="@+id/root_nav_host_fragment"
                 android:name="io.github.droidkaigi.confsched2020.ui.CustomNavHostFragment"
@@ -59,6 +60,7 @@
                 app:layout_constraintStart_toStartOf="parent"
                 app:layout_constraintTop_toBottomOf="@id/toolbar"
                 app:navGraph="@navigation/navigation"
+                tools:ignore="FragmentTagUsage"
                 tools:layout="@layout/fragment_sessions"
                 />
         </androidx.constraintlayout.widget.ConstraintLayout>

